subcommand/sync-catalog/command.go

```diff
@@ -87,12 +95,18 @@ func (c *Command) Run(args []string) int {
 		return 1
 	}
 
+	pollInterval := c.flagAWSPollInterval
+	if pollInterval == DefaultPollInterval && c.flagAWSDeprecatedPullInterval != DefaultPollInterval {
```

Comment:
I realized the second check is not necessary, because it doesn't matter if it is default too..